config/gen/systemd.py

```diff
@@ -78,6 +84,15 @@ class ELECTRONICS_STACK(Enum):
     STACK_2 = '2'
     STACK_3 = '2'
 
+# Set the arduino type based on the argument
+# Used to set the serial port device
+if args.arduino_type == 'spi':
+    jaia_arduino_type=ARDUINO_TYPE.SPI
```

Comment:
Is there a reason for not adding spaces around the equal sign? Is it because it is used in setting the environment variable?

Reply:
No reason, original file style.